addon/adapters/osf-adapter.js

```diff
@@ -39,11 +44,11 @@ export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
         if (relationMeta.options.serializer) {
             serialized = relationMeta.options.serializer(snapshot.record);
         } else {
-            var serializer = store.serializerFor(relationType);
+            var serializer = store.serializerFor(relationType.substring(0, relationType.length - 1));
             if (relationMeta.kind === 'hasMany') {
                 // A hack, since we'd have to use a bulk requests to send a list; TODO remove [0]
-                serialized = snapshot.hasMany(relationship).filter(record => record.id === null).map(record => serializer.serialize(record))[0];
-                delete serialized.data.relationships;
+                var relationArray = snapshot.hasMany(relationship);
+                serialized = serializer.serialize(relationArray[relationArray.length - 1]);
```

Comment:
There's also a delete of the relationships in osf-serializer.  I looked to me like the correct serializer wasn't getting pulled because relationType was plural.  If the correct serializer was used, then there wasn't a need to delete the relationships here.